modules/candidate_list/php/NDB_Menu_Filter_candidate_list.class.inc

```diff
@@ -126,7 +127,9 @@ class NDB_Menu_Filter_Candidate_List extends NDB_Menu_Filter
                 ON (ps.participant_status=pso.ID)
             LEFT JOIN Project p ON (c.ProjectID=p.ProjectID)
             LEFT JOIN subproject sp ON (s.SubprojectID=sp.SubprojectID)
-            WHERE c.Entity_type = 'Human' AND c.Active = 'Y'";
+            WHERE (c.Entity_type = 'Scanner' OR c.Entity_type = 'Human') 
```

Comment:
I think @Jkat is right. The field is defined as `enum('Human', 'Scanner') NOT NULL` in the schema.. so this clause is meaningless.